components/Results/results.tsx

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable @next/next/no-img-element */
+import { useEffect, useState } from "react"
+import { LensShareButton } from "@infinity-keys/react-lens-share-button"
+import "@infinity-keys/react-lens-share-button/dist/style.css"
+import Confetti from "react-confetti"
+import SkeletonCard from "../SkeletonCard"
+import getIpfsLink from "../../lib/getIpfsLink"
+import useWindowSize from "../../lib/useWindowSize"
+
+const Results = ({ metadata }: any) => {
```

Comment:
:nit is any type necessary? it really adds zero value to readability

Reply:
```{
image:string
name: string
animationUrl: string
tokenId: string
}```

Reply:
yeah, type is required